unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/SkyboxCamera.cs

```diff
@@ -24,6 +24,11 @@ public SkyboxCamera()
 
             var cameraData = skyboxCamera.GetUniversalAdditionalCameraData();
             cameraData.renderShadows = false;
+
+            // This index is defined in UniversalRenderPipelineAsset
+            // We are using a custom ForwardRenderer with less features that increase the performance and lowers the passes
+            cameraData.SetRenderer(1);
```

Comment:
Cannot you specify manually the renderer reference in the prefab?

Reply:
There is no prefab, this class creates the camera game object, also the SetRenderer only works with an index 😓

Reply:
oh shit, resolving hahaha